lib/playground.dart

```diff
@@ -187,7 +191,12 @@ class Playground {
 
     keys.bind('ctrl-s', _handleSave);
     keys.bind('ctrl-enter', _handleRun);
-    keys.bind('f1', _handleHelp);
+    keys.bind('f1', () => _toggleDocTab());
+    document.onKeyUp.listen((e) => _handleHelp());
+    document.onClick.listen((e) => _handleHelp());
```

Comment:
And, a mouse click should probably only query if:
- the dart tab is active
- the documentation tab is active